docs/css/buttons.css

```diff
@@ -0,0 +1,49 @@
+a.xournalppButton {
+    background-color: var(--md-primary-fg-color);
+    color: var(--md-primary-bg-color);
+    padding: 10px;
+    cursor: pointer;
+    margin: 8px;
+    font-size: 20px;
+}
+
+a.xournalppButton:hover {
+    color: var(--md-primary-bg-color--light) ;
+}
+
+a.bigXournalppButton {
+    background-color: var(--md-primary-fg-color);
+    color: var(--md-primary-bg-color);
+    padding: 10px;
+    cursor: pointer;
+    margin: 8px;
+    font-size: 24px;
+}
+
+a.bigXournalppButton:hover {
+    color: var(--md-primary-bg-color--light);
+}
+
+#osContainer {
```

Comment:
Refactor the container styles into single class `.download-container` to keep the overall style consistent, and then attach OS-specific classes if needed (e.g. to define the template columns).